package.json

```diff
@@ -17,17 +17,17 @@
     "@carbon/styles": "1.x"
   },
   "dependencies": {
-    "@angular-devkit/core": "^17.3.8",
+    "@angular-devkit/core": "^18.2.12",
     "@angular-extensions/elements": "^17.1.1",
-    "@angular/animations": "^17.3.12",
+    "@angular/animations": "^18.2.13",
     "@angular/cdk": "^17.3.10",
```

Comment:
Can we bump `@angular/cdk` and `@angular-extensions/elements` to the latest `18.2.x.` releases as well?